static/panes/output.ts

```diff
@@ -178,11 +178,7 @@ export class Output extends Pane<OutputState> {
             if (obj.text === '') {
                 this.add('<br/>');
             } else {
-                this.add(
-                    this.normalAnsiToHtml.toHtml(obj.text),
-                    lineNumber,
-                    columnNumber,obj.tag?.file
-                );
+                this.add(this.normalAnsiToHtml.toHtml(obj.text), lineNumber, columnNumber, obj.tag?.file);
```

Comment:
unrelated?

Reply:
Unrelated - I ran prettier

Reply:
Got it! As you should! (though stupid precommits....) anyway, I'll check in the prettier first then you can merge...